pkgs/top-level/all-packages.nix

```diff
@@ -7697,7 +7697,7 @@ in
   inherit (beam.interpreters)
     erlang erlangR18 erlangR19 erlangR20 erlangR21
     erlang_odbc erlang_javac erlang_odbc_javac erlang_nox erlang_basho_R16B02
-    elixir elixir_1_7 elixir_1_6 elixir_1_5 elixir_1_4 elixir_1_3
+    elixir elixir_1_8 elixir_1_7 elixir_1_6 elixir_1_5 elixir_1_4 elixir_1_3
```

Comment:
Can we remove some older ones?

Reply:
Given [the official documentation](https://hexdocs.pm/elixir/1.8.0-rc.1/compatibility-and-deprecations.html), Elixir 1.3 is indeed not supported anymore. I’ll push a new commit to drop support in `nixpkgs` tonight.

Reply:
@Mic92 I forgot to do it yesterday evening, but now it is done.